docs/navigation-advanced.md

```diff
@@ -0,0 +1,399 @@
+# Advanced App navigation
+This documentation describes the steps to modify the NavigationService to allow navigation in different frames and  navigation levels to support advanced navigation scenarios, like the following:
+
+**App launching**
+ - Show a Startup page on app launching and navigate to a navigation pane shell page from there. 
+ - Give the possibility to go back to the startup page or not.
+ 
+**NavigationPane Scenarios**
+ - Navigate among the navigation pane pages with or without the possibility to navigate back using the back button.
+ - Navigate to a page in full screen mode.
+ - Navigate to a logout page that goes back to the startup page and resets the navigation.
+ 
+## NavigationService
+The NavigationService allows you to handle different navigation levels and track the the whole navigation stack in different frames to go back across the navigation tree.
+
+You need to add this code replacing the current NavigationService class.
+```csharp
+public static class NavigationService
+{
+    public const string FrameKeyMain = "Main";
```

Comment:
should this be public? Isn't the only permitted use within `InitializeMainFrame`?